crates/flux-fhir-analysis/src/wf/param_usage.rs

```diff
@@ -110,9 +110,10 @@ impl<'a, 'genv, 'tcx> ParamUsesChecker<'a, 'genv, 'tcx> {
                     self.errors.emit(InvalidParamPos::new(var.span, sort));
                 }
             }
-            fhir::ExprKind::Abs(_, body) => {
+            fhir::ExprKind::Abs(_, body) | fhir::ExprKind::BoundedQuant(_, _, _, _, body) => {
                 self.check_func_params_uses(body, true, is_top_level_var);
```

Comment:
This would allow horn generic refinements in negative positions

```rust
use flux_rs::attrs::*;

#[sig(fn[hrn p: int -> bool](x: i32{ exists i in 0..10 { p(i) } }))]
fn foo(x: i32) { }
```

Reply:
not sure I quite follow, but I replaced with the same flags as `Neg` ... ?

Reply:
Yes, lets just stick with the latter for now!